docs/docs/core/useAnimatedStyle.mdx

```diff
@@ -128,3 +132,24 @@ const styles = StyleSheet.create({
 | ✅      | ✅  | ✅  |
 
 </div>
+
+## Advanced
```

Comment:
Can we put this in Remarks so we won't have a couple sections that serve the same purpose?

```suggestion

```